src/components/FilterSearch.tsx

```diff
@@ -93,9 +96,11 @@ export function FilterSearch({
     if (filter && displayName) {
       answersActions.setFilterOption({ ...filter, displayName, selected: true });
       answersActions.setOffset(0);
-      executeSearch(answersActions);
+      if (searchOnSelect) {
```

Comment:
in the case where `searchOnSelect` is false, how will the user know which filter will be applied when they click an apply/search button? currently, the search input will update to show the filter value when it's selected. but, the user could start typing in the input again to look for other filters. in this case, should the previously selected filter be removed from headless? or do we want to display it in some way so the user knows what has already been selected and will be applied to the next search?

Reply:
Product is ok with this! They referenced zocdoc as having this exact behavior and it feeels good to me? I guess because in my brain if you have already selected a filter you're unlikely to go back and select a different one. Even if you did you probably wouldn't type a different value, then decide not to select a new value, and THEN hit the search button. Even if you did it doesn't seem terribly jank?
https://yext.slack.com/archives/C032CKFARGS/p1655220249371579